lib/bolt/shell/powershell.rb

```diff
@@ -300,6 +300,12 @@ def execute(command)
         stderr.join
         result.exit_code = t.value.respond_to?(:exitstatus) ? t.value.exitstatus : t.value
 
+        if result.exit_code == 126
```

Comment:
I don't think this applies to powershell

Reply:
D'oh. Thanks Nick